languages/elm/maintainers.md

```diff
@@ -0,0 +1,11 @@
+# Elm Maintainers
+
+These awesome people help maintain the Elm track.
+
+## Senior Maintainers
+
+TODO: add senior maintainers
+
+## Contributing Maintainers
+
+Cedd Burge
```

Comment:
The format used in the other tracks is slightly different: https://github.com/exercism/v3/blob/master/languages/csharp/maintainers.md Maybe update to match this?

Reply:
Hi Erik, I've updated this to match the c# format